tests/Aspire.Workload.Tests/StarterTemplateRunTestsBase.cs

```diff
@@ -112,22 +140,17 @@ static async Task<List<string[]>> GetAndValidateCellTexts(ILocator tableLoc)
 
     public static List<ResourceRow> GetExpectedResources(AspireProject project, bool hasRedisCache)
     {
-        List<ResourceRow> expectedResources = new()
+        var expectedResources = new List<ResourceRow>
         {
-            new ResourceRow(
-                Type: "Project",
+            new(Type: "Project",
                 Name: "apiservice",
                 State: "Running",
                 Source: $"{project.Id}.ApiService.csproj",
-                Endpoints: ["http://localhost:\\d+", "https://localhost:\\d+"]),
-
-            new ResourceRow(
-                Type: "Project",
-                Name: "webfrontend",
```

Comment:
The test will fail because it won't recognize the `webfrontend` resource that it would find on the dashboard.

Reply:
I might have to sync up with you in a call. Sorry, first time seeing these tests and I'm struggling following how they're all structured.

Reply:
I can fix the tests a bit for you, if you want.